packages/strapi-design-system/src/Field/FieldLabel.js

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
+import { Flex } from '../Flex';
 import { useField } from './FieldContext';
 import { Typography } from '../Typography';
 
 const TypographyAsterisk = styled(Typography)`
   line-height: 0;
 `;
 
-export const FieldLabel = ({ children, required, ...props }) => {
+const Action = styled(Flex)`
+  line-height: 1;
```

Comment:
Good catch! I think the tooltip causes this issue of misalignment. 
I tried `line-height: 0` as for TypographyAsterisk and it seems to be even better aligned, not a fan of those tricks but not sure how to handle this, WDYT? @gu-stav 

<img width="643" alt="image" src="https://user-images.githubusercontent.com/71838159/154483661-663f9021-f4a9-4414-b6c1-f392c6a37afc.png">

Reply:
Me neither, but I think it this case it gets the job done 🤓